<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QWidget" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="VolumeSlider">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>130</y>
     <width>16</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="VedioSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlayerUrl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PauseButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QWidget" name="VideoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>821</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="RateComboBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>520</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0.5x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.0x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.0x</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>视频播放地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimerEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>480</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00/00:00:00</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
